codex-ui/src/vue/components/alert/constant.ts

```diff
@@ -0,0 +1,15 @@
+import type { AlertOptions } from './Alert.types';
+import { POSITION } from './Alert.types';
+
+/**
+ * default styles
+ */
+const ALERT_CONTAINER_STYLES: AlertOptions = {
```

Comment:
description and variable name are confusing since it does not contain styles

Reply:
moved to separate files, effected